app/models/patient.rb

```diff
@@ -109,8 +109,10 @@ class Patient
             :line,
             presence: true
   validates :primary_phone, format: /\d{10}/,
-                            length: { is: 10 },
-                            uniqueness: true
+                            length: { is: 10 }
+
+  validate :confirm_unique_phone_number, on: :create
```

Comment:
Should do this on all state changes, not just create -- thoughts?

Reply:
You're right. Turns out I wrote my custom validator wrong. 

I set it to `on: :create` b/c the database seeding script kept yelling at me. My custom validator works for creating new items, but fails for updating existing items.

Say you wanted to do this to your data base:
```
- name: 'John', primary_phone: '123-456-7890'
+ name: 'Robert', primary_phone: '123-456-7890
```

You'd probably do this by searching for the John item, and then updating the name. Problem is, my custom validator sees your proposed change as a duplicate number. It doesn't know what an update is.

I need to change the validation function to handle updates. Uniqueness checking is trickiness than I first thought.

Reply:
done